### ISSUE TITLE
Não é possível aceder a este site para download das versões >118

### ISSUE THREAD

**Issue by nathaliahenriques** [GoogleChromeLabs/chrome-for-testing#74]
![image](https://github.com/GoogleChromeLabs/chrome-for-testing/assets/139583548/8820f327-ce27-453c-82c3-881bd2c21f15)

**Comment by danexplore** [comment]
estou com o mesmo problema

**Comment by Anic10x3** [comment]
Facing the same issue, The Automation script is not able to download the webdriver. Blocked due to this.

**Comment by thiagowfx** [comment]
This seems like a duplicate of https://github.com/GoogleChromeLabs/chrome-for-testing/issues/60

**Comment by thiagowfx** [comment]
Closing on the grounds that it's likely a duplicate of https://github.com/GoogleChromeLabs/chrome-for-testing/issues/60

If it turns out it's not, and/or if you're still blocked by this and have more details/logs to help debug it, feel free to reopen it.
